module/VuFind/src/VuFind/RecordDriver/AbstractBase.php

```diff
@@ -246,6 +246,8 @@ public function deleteTags($user, $tags)
      * @param ?int $userId User ID, or null for all users
      *
      * @return array
+     *
+     * @deprecated Use \VuFind\Ratings\RatingsService::getRatingData()
```

Comment:
We can't redirect these methods without adding new record driver dependencies, so I'm just deprecating them as-is. I'd really love to get rid of them and their associated property, but we'll just have to wait until release 11.